jumpscale/packages/vdc_dashboard/bottle/deployments.py

```diff
@@ -186,6 +186,13 @@ def get_zstor_config():
         "redundant_nodes": 0,
         "encryption": {"algorithm": "AES", "key": encryption_key,},
         "compression": {"algorithm": "snappy",},
+        "meta": {
+            "type": "etcd",
+            "config": {
+                "endpoints": ["http://127.0.0.1:2379", "http://127.0.0.1:22379", "http://127.0.0.1:32379"],
```

Comment:
how did you specify these endpoints?

Reply:
It'll be the default values for etcd, can be modified later if the user wants so as the secret and prefix